src/Agent.Worker/TestResults/ResultsCommandExtension.cs

```diff
@@ -28,6 +28,8 @@ public sealed class ResultsCommandExtension : AgentService, IWorkerCommandExtens
 
         public string CommandArea => "results";
 
+        public string SupportedHostType => string.Empty;
```

Comment:
We set Build_RequestedForId/Build_BuildId and not Build_RequestedFor/Build_BuildUri, but test results will still work in RM.